### PR TITLE
Initial support for GitHub actions

### PR DESCRIPTION
**Pull request by turekj**
- [x] Initial support for Github Actions
- [x] Self-linting with `danger-python`

### CODE REVIEW

**Comment by danger-public** [comment]
[WIP] Support for GitHub actions
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/python">danger-python</a> against c93d7675bda5abd4c7af62cff0ff469ee559a240
</p>

File: .github/workflows/danger.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run Danger
+      uses: ./
```
Comment (orta) [inline .github/workflows/danger.yml]: does this work?

Comment (turekj) [reply .github/workflows/danger.yml]: It sure does! 😸 I'll add the usage to the README

https://github.com/turekj/danger-py-ga-test/pull/1

**Comment by danger-public** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  danger-python` failed.
  </td>
    </tr>
  </tbody>
</table>



### Log


```sh

```

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against d4f839794a4e6217f34cfac8cf3b04ac58a56f0f
</p>

**Pull request merged**